problemsets-mat223/linearalbegra.tex

```diff
@@ -5007,9 +5007,27 @@ \section*{Determinants}
 	Suppose you know $\det(X)=4$.
 	\begin{parts}
 		\item What is $\det(X^{-1})$?
+			\begin{solution}
+				$\det(X^{-1})=\frac{1}{4}$.
+
+				We know that $XX^{-1}=I$. Therefore we must have that 
+				$\det(XX^{-1})=\det(X)\det(X^{-1})=\det(I)=1$, 
+				and so $\det(X^{-1})=\frac{1}{4}$.
+			\end{solution}
 		\item Derive a relationship between $\det(Y)$
 			and $\det(Y^{-1})$ for an arbitrary matrix $Y$.
-		\item Suppose $Y$ is not invertible.  What is $\det(Y)$?
+			\begin{solution}
+				$\det(Y^{-1})=\frac{1}{\det(Y)}$.
+
+				Using the same reasoning as the previous part, we know that
+				$YY^{-1}=I$. Therefore we must have 
+				$\det(Y)\det(Y^{-1})=\det(YY^{-1})=\det(I)=1$, and so 
+				$\det(Y^{-1})=\frac{1}{\det(Y)}$.
+			\end{solution}
+		\item Suppose $Y$ is not invertible. What is $\det(Y)$?
+			\begin{solution}
+				$\det(Y)=0$. 
```

Comment:
A bit more here. "This makes sense because for a square matrix `X`, `\det(X^{-1}) = 1/\det(X)`. This formula always works, except when `\det(X)=0`."